docs/v0.7/en/greptimecloud/usage-&-billing/wcu-rcu.md

```diff
@@ -0,0 +1,89 @@
+# WCU & RCU
+
+This document will introduce the usage calculation algorithms of GreptimeCloud. To monitor service usage, you can go to the [GreptimeCloud Console](https://console.greptime.cloud/).
+
+
+## Capacity unit
+
+All requests to GreptimeCloud are measured in capacity units, which reflect the size and complexity of the request. The measurement methods of write capacity unit and read capacity unit are different, see following for details.
+
+### WCU (Write Capacity Unit)
+
+Each API call to write data to your table is a write request.
+WCU is calculated based on the total size of the insert rows in one request.
+A standard write capacity unit can write rows up to 1KB.
+For rows larger than 1KB, additional write capacity units are required.
+
+:::tip NOTE
+The capacity unit may be subject to change in the future.
+:::
+
+The following steps are used to determine the size of each request:
+
+1. Get the size of the data type of each column in the table schema. You can find details about the size of each data type in the [Data Types](/reference/sql/data-types.md) documentation.
+2. Sum up the sizes of all columns in the request. If a column is not present in the request, its size depends on the column's default value. If the default value is null, the size is 0; otherwise, it is the size of the data type.
+3. Multiply the sum by the number of rows to be written.
+
+Here's an example of how to calculate the WCU for a table with the following schema:
+
+```shell
++-------------+----------------------+------+------+---------------------+---------------+
+| Column      | Type                 | Key  | Null | Default             | Semantic Type |
++-------------+----------------------+------+------+---------------------+---------------+
+| host        | String               | PRI  | YES  |                     | TAG           |
+| idc         | String               | PRI  | YES  |                     | TAG           |
+| cpu_util    | Float64              |      | YES  |                     | FIELD         |
+| memory_util | Float64              |      | YES  |                     | FIELD         |
+| disk_util   | Float64              |      | YES  |                     | FIELD         |
+| ts          | TimestampMillisecond | PRI  | NO   | current_timestamp() | TIMESTAMP     |
++-------------+----------------------+------+------+---------------------+---------------+
+```
+
+You have a write request as following:
+
+```shell
+INSERT INTO system_metrics VALUES ("host1", "a", 11.8, 10.3, 10.3, 1667446797450);
+```
+
+Based on the size of the data types in your table schema, the size of each row is 38 bytes (5+1+8+8+8+8), and the WCU of this request is 1 according to the calculation algorithm.
```

Comment:
> the size of each row is 38 bytes (5+1+8+8+8+8), and the WCU of this request is 1 according to the calculation algorithm.

I see no `round` or `ceiling` is explicitly said. Should we mention that here or in the calculation algorithm?